### PR TITLE
Update quickstart-jailbreak.md

### PR DESCRIPTION
**Pull request by jinruishao**

### CODE REVIEW

**Comment by PatrickFarley** [comment]
#sign-off

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 1]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
 
 # Quickstart: Prompt Shields (preview)
 
-Follow this guide to use Azure AI Content Safety Prompt Shields to check your large language model (LLM) inputs for both User Prompt and Document attacks.
+## Introduction
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 2]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- To identify and mitigate document attacks where harmful content is embedded within user-provided documents.
+- To maintain the integrity, safety, and compliance of AI-generated content, thereby preventing misuse of GenAI systems.
 
 For more information on Prompt Shields, see the [Prompt Shields concept page](./concepts/jailbreak-detection.md). For API input limits, see the [Input requirements](./overview.md#input-requirements) section of the Overview. 
 
+
+
+
+## User Scenarios
+### 1. AI Content Creation Platforms: Detecting Harmful Prompts
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
## User scenarios
### 1. AI content creation platforms: Detecting harmful prompts
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 3]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- Scenario: An AI content creation platform uses generative AI models to produce marketing copy, social media posts, and articles based on user-provided prompts. To prevent the generation of harmful or inappropriate content, the platform integrates "Prompt Shields."
+- User: Content creators, platform administrators, and compliance officers.
+- Action: The platform uses Azure AI Content Safety's "Prompt Shields" to analyze user prompts before generating content. If a prompt is detected as potentially harmful or likely to lead to policy-violating outputs (e.g., prompts asking for defamatory content or hate speech), the shield blocks the prompt and alerts the user to modify their input.
+- Outcome: The platform ensures all AI-generated content is safe, ethical, and compliant with community guidelines, enhancing user trust and protecting the platform's reputation.
+### 2. AI-Powered Chatbots: Mitigating Risk from User Prompt Attacks
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
### 2. AI-powered chatbots: Mitigating risk from user prompt attacks
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 4]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- Scenario: A customer service provider uses AI-powered chatbots for automated support. To safeguard against user prompts that could lead the AI to generate inappropriate or unsafe responses, the provider uses "Prompt Shields."
+- User: Customer service agents, chatbot developers, and compliance teams.
+- Action: The chatbot system integrates "Prompt Shields" to monitor and evaluate user inputs in real-time. If a user prompt is identified as potentially harmful or designed to exploit the AI (e.g., attempting to provoke inappropriate responses or extract sensitive information), the shield intervenes by blocking the response or redirecting the query to a human agent.
+- Outcome: The customer service provider maintains high standards of interaction safety and compliance, preventing the chatbot from generating responses that could harm users or breach policies.
+### 3. E-Learning Platforms: Preventing Inappropriate AI-Generated Educational Content
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
### 3. E-learning platforms: Preventing inappropriate AI-generated educational content
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 5]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- Scenario: An e-learning platform employs GenAI to generate personalized educational content based on student inputs and reference documents. To avoid generating inappropriate or misleading educational content, the platform utilizes "Prompt Shields."
+- User: Educators, content developers, and compliance officers.
+- Action: The platform uses "Prompt Shields" to analyze both user prompts and uploaded documents for content that could lead to unsafe or policy-violating AI outputs. If a prompt or document is detected as likely to generate inappropriate educational content, the shield blocks it and suggests alternative, safe inputs.
+- Outcome: The platform ensures that all AI-generated educational materials are appropriate and compliant with academic standards, fostering a safe and effective learning environment.
+### 4. Healthcare AI Assistants: Blocking Unsafe Prompts and Document Inputs
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
### 4. Healthcare AI assistants: Blocking unsafe prompts and document inputs
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 6]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- Scenario: A healthcare provider uses AI assistants to offer preliminary medical advice based on user inputs and uploaded medical documents. To ensure the AI does not generate unsafe or misleading medical advice, the provider implements "Prompt Shields."
+- User: Healthcare providers, AI developers, and compliance teams.
+Action: The AI assistant employs "Prompt Shields" to analyze patient prompts and uploaded medical documents for harmful or misleading content. If a prompt or document is identified as potentially leading to unsafe medical advice, the shield prevents the AI from generating a response and redirects the patient to a human healthcare professional.
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
- Action: The AI assistant employs "Prompt Shields" to analyze patient prompts and uploaded medical documents for harmful or misleading content. If a prompt or document is identified as potentially leading to unsafe medical advice, the shield prevents the AI from generating a response and redirects the patient to a human healthcare professional.
```

File: articles/ai-services/content-safety/quickstart-jailbreak.md [hunk 7]
```diff
@@ -13,10 +13,47 @@ ms.author: pafarley
...
+- Outcome: The healthcare provider ensures that AI-generated medical advice remains safe and accurate, protecting patient safety and maintaining compliance with healthcare regulations.
+### 5. Generative AI for Creative Writing: Protecting Against Prompt Manipulation
```
Comment (Court72) [inline articles/ai-services/content-safety/quickstart-jailbreak.md]:
```suggestion
### 5. Generative AI for creative writing: Protecting against prompt manipulation
```

**Comment by Court72** [comment]
@PatrickFarley - The article **articles/ai-services/content-safety/quickstart-jailbreak.md** might benefit from a copy edit. An editor will notify the listed author and recent contributors.

**Pull request merged**